src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -310,6 +356,160 @@ internal override void SetDefaultSourcePath(string defaultPath)
             WriteValueToFile<string>(fileName, "DefaultSourcePath", defaultPath);
         }
 
+#if UNIX
+        /// <summary>
+        /// Gets the identity name to use for writing to syslog.
+        /// </summary>
+        /// <returns>
+        /// The string identity to use for writing to syslog.
+        /// <para>
+        /// The default value is 'powershell'
+        /// </para>
+        /// </returns>
+        internal override string GetSysLogIdentity()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string identity = ReadValueFromFile<string>(fileName, "LogIdentity");
+            if
+            (
+                string.IsNullOrEmpty(identity)
+                ||
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, identity) == 0
+            )
+            {
+                identity = "powershell";
+            }
+            return identity;
+        }
+
+        /// <summary>
+        /// Gets the log level filter.
+        /// </summary>
+        /// <returns>One of the PSLevel values indicating the level to log.
+        /// <para>
+        /// The default value is PSLevel.Informational
+        /// </para>
+        /// </returns>
+        internal override PSLevel GetLogLevel()
+        {
+            string fileName = Path.Combine(psHomeConfigDirectory, configFileName);
+            string levelName = ReadValueFromFile<string>(fileName, "LogLevel");
+            PSLevel level;
+            if
+            (
+                StringComparer.OrdinalIgnoreCase.Compare(LogDefaultValueName, levelName) == 0
```

Comment:
[Above](https://github.com/PowerShell/PowerShell/pull/5144/files#r145826931) we decided to use `Equals()`.

Reply:
Yes we did; let me make sure it got pushed.

Reply:
fixed